sqlglot/dialects/prql.py

```diff
@@ -103,6 +104,16 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_order_by(self, query: exp.Select) -> t.Optional[exp.Query]:
+            order = self._parse_sort(exp.Order, TokenType.L_BRACE)
+            if order and not self._match(TokenType.R_BRACE, expression=query):
+                self.raise_error("Expecting }")
+            return (
+                query.order_by(order)
+                if order
+                else query.order_by(self._parse_order(skip_order_token=True))
```

Comment:
copy=False everywhere

Reply:
i don't think you need parse_order here, you can just do 

self.expression(exp.Order, expressions=self._parse_csv(self._parse_ordered))